hastory-server-data/src/Hastory/Server/Data/UserForm.hs

```diff
@@ -6,15 +6,15 @@ module Hastory.Server.Data.UserForm where
 import Data.Aeson
 import qualified Data.Text as T
 import Data.Validity
+import Data.Validity.Text ()
 import GHC.Generics
 
-import Hastory.Server.Data.Password
 import Hastory.Server.Data.Username
 
 data UserForm =
   UserForm
     { userFormUserName :: Username
-    , userFormPassword :: Password
+    , userFormPassword :: T.Text
```

Comment:
`T.Text` is really ugly.
Instead, import Text twice like this:
```
import qualified Data.Text as T
import Data.Text (Text)
```

Reply:
Done.